README.md

```diff
@@ -2,16 +2,25 @@
 
 This repository contains the default official module repository for [BlueBuild](https://blue-build.org/). See [./modules/](./modules/) for the module source code. See the [website](https://blue-build.org/reference/module/) for module documentation. See [How to make a custom module](https://blue-build.org/how-to/making-modules/) and [Contributing](https://blue-build.org/learn/contributing/) for help with making custom modules and contributing.
 
-## Style guidelines for official modules
+## Style & code guidelines for official bash modules
 
-These are general guidelines for writing official modules and their documentation to follow to keep a consistent style. Not all of these are to be mindlessly followed, especially the ones about grammar and writing style, but it's good to keep these in mind if you intend to contribute back upstream, so that your module doesn't feel out of place.
+These are general guidelines for writing official bash modules and their documentation to follow in order to keep a consistent style. Not all of these are to be mindlessly followed, especially the ones about grammar and writing style, but it's good to keep these in mind if you intend to contribute back upstream, so that your module doesn't feel out of place.
 
-### Bash
+### Terms
+
+**Module-maintainer:** Maintainer of the BlueBuild bash module, to whom is this guideline targeted.  
+**Image-maintainer:** Maintainer of the custom image which uses BlueBuild template.  
+**Local-user:** User which uses a custom image, which utilizes BlueBuild bash modules. Image-maintainer can be a local-user too at the same time.
+
+### Code Rules
 
 - Echo what you're doing on each step and on errors to help debugging.
-- Don't echo blocks using "===", that is reserved for the code launching the modules.
+- Implement error-checks for scenarios where image-maintainer can make a mistake.
 - Use `snake_case` for functions and variables changed by the code.
 - Use `SCREAMING_SNAKE_CASE` for variables that are set once and stay unchanged.
+- Use `"${snake_case}"` to ensure that variables are properly parsed for strings.
```

Comment:
When should this be used? This could be rewritten to be in the form of "Use `"${variable_name}"` when xyz to ensure that variables are properly parsed as strings."

Reply:
I rewrote this, tell me if it's good enough, or it needs to be even better

Reply:
LGTM